### ISSUE TITLE
209A: What if same name 

### ISSUE THREAD

**Issue by kateleebarry** [SuffolkLITLab/docassemble-MAVirtualCourt#262]
I'm going through the interview trying to  come up with a list of screens/questions that could be ambiguous if parties and/or children have the same name: 

Here, maybe change "them" to "the children": 
![image](https://user-images.githubusercontent.com/63356951/82471137-4b74bb00-9a94-11ea-8245-e9db84a358e7.png)

This one I could see being an issue if a parent/child have the same first and last name.  Maybe "Where does your child, [child name], live?"
![image](https://user-images.githubusercontent.com/63356951/82471863-4e23e000-9a95-11ea-96b0-ba4f083a7c68.png)

Maybe, "Has your child, [child name] lived anywhere else..."
![image](https://user-images.githubusercontent.com/63356951/82471901-5aa83880-9a95-11ea-9ea4-cd5382091cf1.png)

Same as above
![image](https://user-images.githubusercontent.com/63356951/82471979-77dd0700-9a95-11ea-8121-5f5ef0d96fd1.png)

Maybe "Defendant Information"
![image](https://user-images.githubusercontent.com/63356951/82472108-a1962e00-9a95-11ea-8305-9b6b1560883a.png)

Will add more after standup

**Comment by kateleebarry** [comment]
![image](https://user-images.githubusercontent.com/63356951/82472275-efab3180-9a95-11ea-9e84-04a96c95614d.png)

**Comment by kateleebarry** [comment]
![image](https://user-images.githubusercontent.com/63356951/82472840-c048f480-9a96-11ea-9f94-9be3c6ff1773.png)

**Comment by kateleebarry** [comment]
Pretty much all the defendant information screens, I'm going to stop pointing to each individual screen.

**Comment by kateleebarry** [comment]
okay, that's it

**Comment by nonprofittechy** [comment]
This is a good list but it may be clear enough from context for some of these. Maybe @CaroRob take a look with ambiguous names and think of whether we need any more context clues. Section labels may help too

**Comment by kateleebarry** [comment]
I agree some of these might be clear enough from context. The big reason I erred on the side of being over-inclusive was with the thought that people might leave and return to their interview later or take a break and forget the previous questions, so I thought it might be good if each individual question could stand clearly on its own without the context of the previous questions?

**Comment by CaroRob** [comment]
I kinda hate to do this, I get why it's important to make the distinction between child and parent names,  however the whole child/parent as part of person we refer to/name in a question interferes significantly with readability. 
Bearing in mind we also anticipate translation complexities, I think this issue is also wrapped up in asking for defendant gender when asking for defendant name -  - thinking ahead to translation, pronouns, verb agreement etc. 
Do we have a method/function for listing children by first name?  ie. Why do you need the court to order Chris to have no contact with Prima, Secundus, Tertia, and Quartus? or the children -

Another possibility - which might solve this unusual but not unheard of issue.
```
if ${other_parties.familiar()} == ${children[i].familiar()}:
  other_parties.familiar() = other_parent_nick
  children_of_both[i] = child_nick
`


question: |
  Looks like ${other_parties.familiar()} and your child, ${children[i].familiar() have the same first name!
subquestion: |
   We are going to have to go through a number of questions when we refer to one or both of them in the same sentence or question. What should we call each of them so we are clear about who we are talking about?
fields:
  - 'nickname for the ${ other_parties.familiar() } you are taking to court: other_parent_nick
  - 'nickname for your child ${ children[i].familiar() }: child_nick
```
Just a thought, because this co-occurrence is going to be somewhat irregular it would be a shame to confuse people or add to reading barriers by adding extra words and kind of confuse people by reminding them of their relationship with their child or defendant every time we mention them.

I think we do not need this for the MVP.
Also not for the MVP -  I think it would be cool to implement something like Brett Harrison used to do in A2J author interviews, wherein whenever you left and saved an interview - the interview set a flag. This way you can return to a certain reliable point and without having to start all over. Also this would be a perfect example of a place for review screens - when you resume your interview - if you have saved it - you begin with the most recent review screen. This would serve as a reminder for where you were in the process, what you said, and how to carry on? Perhaps too much for one github issue - but stream of consciousness.
